orchagent/orch.cpp

```diff
@@ -789,7 +787,7 @@ void Orch::addConsumer(DBConnector *db, string tableName, int pri)
     }
     else
     {
-        addExecutor(new Consumer(new ConsumerStateTable(db, tableName, gBatchSize, pri), this, tableName));
+        addExecutor(new Consumer(new ConsumerStateTable(db, tableName, gBatchSize * CONSUMER_POP_MAX_BATCH_COUNT, pri), this, tableName));
```

Comment:
This change will cause an issue:

https://github.com/sonic-net/sonic-buildimage/commit/286ec3edbf91ec1bd09b6eae3a4d69801bdd94a9


Background running lua script may cause redis-server quite busy if batch size is 8192.
If handling time exceeded default 5s, the redis-server will not response to other process and will cause syncd crash.

So that's also part of reason why add a loop in execute()